### DISCUSSION TITLE
Ultradian rhythm alarm clock

### DISCUSSION THREAD

**Discussion by mayerk1** [zepp-health/zeppos-forum#235] [General]
Loving my Amazfit product but wish it had an ultradian rhythm alarm clock to wake me at the end of a cycle. 
Can't find one anywhere, any chance this would be a future feature?

**Comment by silver-zepp** [comment]
Right now I don't think we have one, but this should be achievable with `ZeppOS 3.0` so I think we'll get it sooner or later.

**Comment by mayerk1** [comment]
Thanks! Any idea when 3.0 is coming out?

**Comment by silver-zepp** [comment]
No problem @mayerk1! We don't have a specific date yet, but the closed beta just ended, so it will be announced soon.
